hat/conf/application.conf

```diff
@@ -324,4 +330,9 @@ hatters {
   scheme = "https://"
 }
 
+
+play.filters.hosts {
+  allowed = ["."]
```

Comment:
Should not be in the main `application.conf` file as might get accidentally deployed.